bigbluebutton-html5/imports/ui/components/audio/service.js

```diff
@@ -23,13 +23,15 @@ const init = () => {
   AudioManager.init(userData);
 };
 
-const exitAudio = () => AudioManager.exitAudio();
-const joinListenOnly = () => AudioManager.joinAudio(true);
-const joinMicrophone = () => AudioManager.joinAudio(false);
+const exitAudio = () => audioManager.exitAudio();
+const joinListenOnly = () => audioManager.joinAudio(true);
+const joinMicrophone = () => audioManager.joinAudio(false);
+const sendCallParameters = () => audioManager.sendCallParameters();
```

Comment:
Why have these been changed from `AudioManager` to `audioManager` when it's being imported here as uppercase AudioManager? Where is the lowercase audioManager coming from and why do we need this extra variable?